.github/workflows/fcos-podman-next-build.yml

```diff
@@ -0,0 +1,52 @@
+name: Build FCOS image with packages from rhcontainerbot/podman-next
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  IMAGE_NAME: fcos
+  IMAGE_TAGS: podman-next ${{ github.sha }}
```

Comment:
For your TODO: Consider using these instead/in addition to: https://github.com/opencontainers/image-spec/blob/main/annotations.md  (can be labels too if there is no annotation option).  Esp. `org.opencontainers.image.source`, and `org.opencontainers.image.version` (if possible) and `org.opencontainers.image.revision` (commit SHA) are all really handy to have.

Bonous points for adding a label with a link to the GHA workflow build log :smile: (I don't think there's an OCI standard for that one).

Reply:
`org.opencontainers.image.created` (timestamp) is good too.

Reply:
Ahh, maybe put link to build-log into `org.opencontainers.image.url`.  The description seems like that might fit the purpose.

Reply:
these should be Containerfile LABELs, correct? Also, what's the url supposed to look like? just `https://quay.io/podman/fcos` ?

Reply:
hmm, these labels aren't in the podmanimage Containerfiles either. Can we leave these for later?